### PR TITLE
Docs: Freshdesk documentation

### PR DESCRIPTION
**Pull request by dat-a-man**
### Description

Added documentation for Freshdesk verified source.

### CODE REVIEW

File: docs/website/docs/dlt-ecosystem/verified-sources/freshdesk.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
+---
+title: Freshdesk
+description: dlt verified source for Freshdesk API
+keywords: [freshdesk api, freshdesk verified source, freshdesk]
+---
+import Header from './_source-info-header.md';
+
+# Freshdesk
+
```
Comment (AstrakhantsevaAA) [inline docs/website/docs/dlt-ecosystem/verified-sources/freshdesk.md:9]: you forgot about this
image

File: docs/website/docs/dlt-ecosystem/verified-sources/freshdesk.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+## Setup Guide
+
+### Grab credentials
+
+To obtain your Freshdesk credentials, follow these steps:
+
+1. Log in to your Freshdesk account.
+1. Click on the profile icon to open "Profile Settings".
+1. Copy the API key displayed on the right side.
```
Comment (AstrakhantsevaAA) [inline docs/website/docs/dlt-ecosystem/verified-sources/freshdesk.md:37]: image

**Review by AstrakhantsevaAA — CHANGES_REQUESTED** [review]
Good work! only two small comments

File: docs/website/docs/dlt-ecosystem/verified-sources/freshdesk.md (hunk 1 above)
Comment (AstrakhantsevaAA) [reply docs/website/docs/dlt-ecosystem/verified-sources/freshdesk.md:9]: just add
```
 <Header/>
```

Comment (dat-a-man) [reply docs/website/docs/dlt-ecosystem/verified-sources/freshdesk.md:9]: added

File: docs/website/docs/dlt-ecosystem/verified-sources/freshdesk.md (hunk 2 above)
Comment (dat-a-man) [reply docs/website/docs/dlt-ecosystem/verified-sources/freshdesk.md:37]: Added Alena, thanks!

**Review by AstrakhantsevaAA — APPROVED** [review]

**Pull request merged**